stdlib/public/Concurrency/TaskPrivate.h

```diff
@@ -42,9 +42,12 @@
 namespace swift {
 
 // Set to 1 to enable helpful debug spew to stderr
+// If this is enabled, tests with `swift_task_debug_log` requirement can run.
 #if 0
 #define SWIFT_TASK_DEBUG_LOG(fmt, ...)                                         \
-  fprintf(stderr, "[%lu] " fmt "\n", (unsigned long)_swift_get_thread_id(),    \
+  fprintf(stderr, "[%lu] [%s:%d](%s) " fmt "\n",                               \
+          (unsigned long)_swift_get_thread_id(),                               \
+          __FILE__, __LINE__, __FUNCTION__,                                    \
```

Comment:
I hope no-one minds making this a bit more informative? It is a life-saver at least for me to have this include where the log is coming from... 🙏

Reply:
Great idea.

Reply:
Yay!